dotCMS/src/main/java/com/dotcms/publisher/assets/business/PushedAssetsAPI.java

```diff
@@ -85,7 +85,24 @@
 	 */
 	public PushedAsset getLastPushForAsset(String assetId, String environmentId, String endpointIds)  throws DotDataException;
 
+	/**
+	 * deletes the push assets entries for the given Bundle Id
+	 *
+	 * @param	bundleId	the id of the bundle
+	 * @throws	DotDataException	thrown when an error in the underlying data layer occurs
+	 */
+
+	public void deletePushedAssetsByBundleId(final String bundleId)  throws DotDataException;
+	/**
+	 * returns all the push assets for a given bundleId and EnvironmentId
+	 *
+	 * @param	bundleId	the id of the bundle
+	 * @param	environmentId	the id of the environment
+	 * @throws	DotDataException	thrown when an error in the underlying data layer occurs
+	 */
+
+	public List<PushedAsset> getPushedAssetsByBundleIdAndEnvironmentId(final String bundleId, final String environmentId)  throws DotDataException;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary modifier 'public' on method 'getPushedAssetsByBundleIdAndEnvironmentId': the method is declared in an interface type](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4502981)